01-shapefile-attributes.Rmd

```diff
@@ -133,19 +133,22 @@ extent(x = point)
 
```

Comment:
In my Rmd file this has a bunch in it about viewing the metadata.  Not sure what happened.  My next pull request will have this fixed.